internal/cluster/distribution/eks/eksprovider/workflow/bootstrap_workflow.go

```diff
@@ -0,0 +1,139 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workflow
+
+import (
+	"time"
+
+	"emperror.dev/errors"
+	"github.com/Masterminds/semver/v3"
+	"go.uber.org/cadence"
+	"go.uber.org/cadence/workflow"
+
+	"github.com/banzaicloud/pipeline/internal/cluster/infrastructure/aws/awsworkflow"
+	pkgCadence "github.com/banzaicloud/pipeline/pkg/cadence"
+)
+
+const BootstrapWorkflowName = "eks-bootstrap-flow"
```

Comment:
Coredns install has been implemented in Bootstrap activity originally, but since we are installing 3 addons now I've created a separate activity for installing a single addon. It seems that some take 60, 120 secs to install so it makes sense to run addon create actions in parallel but before making other updates in old Bootstrap action (with update aws-node Daemon for ex.) and also in parallel with node pool creation. I think adding this flow in createInfra flow directly would make it quite hard to understand that's I think it's easier to follow if these are in a separate workflow.